nondjango/storages/storages.py

```diff
@@ -320,21 +320,23 @@ def _file_hash(self, file: File, function='') -> str:
         else:
             return s3_file.e_tag
 
-    def url(self, name: str):
+    def url(self, name: str, check_for_inexistent=False):
```

Comment:
I would default true on this. I assume that is the behavior most people would want.

Reply:
ok.

Reply:
Done on 1e46a04